todo.md

```diff
@@ -1,6 +1,5 @@
 # Todo list
 
-* [ar](#ar)
```

Comment:
You don't use "php script/todo.php"

Reply:
So, you remove this PR and I make another one after using `php script/todo.php` or next time?